app/presenters/tree_builder_ops_rbac_features.rb

```diff
@@ -55,7 +59,7 @@ def tree_init_options
       :three_checks => true,
       :post_check   => true,
       :check_url    => "/ops/rbac_role_field_changed/",
-      :oncheck      => @editable ? "miqOnCheckGeneric" : false
+      :oncheck      => @editable ? "miqOnCheckGeneric" : ""
```

Comment:
Trying to understand this, why an empty string? Can't this be just `nil`?

Reply:
Don't know, most probably yes, false was definitely wrong and caused errors, it doesn't do it now.

Reply:
Well, it was probably causing errors because you had no whitelist and `false` isn't a legit function name. But I'd prefer `nil` over an empty string.